src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1998,25 +1969,6 @@ protected function DeleteRecordReferencesProperties(&$oPropertyField, $bRevision
             $query = 'SELECT * FROM `'.MySqlLegacySupport::getInstance()->real_escape_string($propertyTable).'` WHERE `'.MySqlLegacySupport::getInstance()->real_escape_string($foreignKeyName)."` = '".MySqlLegacySupport::getInstance()->real_escape_string($this->sId)."'";
 
             $sRecordIdWhiteList = '';
-            if ($bRevisionActivationMode && $bAllowRecordReferenceDeletion) {
-                $sTableID = TTools::GetCMSTableId($propertyTable);
-
-                if (array_key_exists(self::DELETE_REFERENCES_REVISION_DATA_WHITELIST_SESSION_VAR, $_SESSION)) {
```

Comment:
The constant should also be deprecated.

Reply:
It is already deprecated.

Reply:
Ah you meant in the upgrade markdown.... added.